engine/src/main/java/io/camunda/zeebe/engine/processing/BpmnProcessors.java

```diff
@@ -236,7 +236,8 @@ private static void addProcessInstanceMigrationStreamProcessors(
             processingState.getElementInstanceState(),
             processingState.getProcessState(),
             processingState.getJobState(),
-            processingState.getVariableState()));
+            processingState.getVariableState(),
+            processingState.getIncidentState()));
```

Comment:
This is growing a bit out of proportion 😆 

🔧 We could simply pass `processingState` to the processor, allowing the constructor to pick the needed parts.

Reply:
Makes total sense. 👍 Updated!